client/packages/system/src/Patient/api/hooks/keys.ts

```diff
@@ -1 +1 @@
-export const VACCINATION_CARD = 'vaccination-card';
+export const VACCINATION = 'vaccination';
```

Comment:
So this key needs be the same as the one in `client/packages/system/src/Vaccination/api/keys.ts` in order to cause the table refresh after vaccination.

Should we refer back to that one so that the Vax Card hook is referencing the same constant? Not sure what pattern we've established when two hooks that share a key are distantly separated in the file structure.

Reply:
I think better would be to keep this as `VACCINATION_CARD`, and add `VACCINATION_CARD` to the invalidate queries call after the vaccination insert mutation?